mysystehis/main.tex

```diff
@@ -0,0 +1,1155 @@
+ \renewcommand{\thesubsection}{\arabic{subsection}}
+\section{\Large List of Abbreviations} 
+\begin{itemize}
+	\item GPU: Graphic processing unit (in graphic card)
+	\item NPC: Non-Playable character
+	\item FPS: First person
+	\item TPS: Third person
+	\item GI: Global illumination	
+	\item RGB: Red, Green and Blue
+\end{itemize}
+\newpage
+\section{\Large Introduction} 
+Nowadays, the technology is getting better and better, so human life are becoming more and more convenient. With that, people's entertainment needs are also increasing. However, to satisfy people's needs, also to keep up with this rapid development of technology, a virtual world is also a type of entertainment that can satisfy people's need. \\[0.35cm]  A virtual world is a computer-based environment that the user can interact with each other in the world. In general, the virtual world usually using 3-dimensional graphics, with 3D models, which can make people feel like they're in the real world. Virtual worlds allow for multi user to communicate, and a 3D video single player game like Elder Scroll: Blade and Skyrim can still be consider as the virtual world. \\[0.35cm] In general term, there's still no generally accepted definition for virtual world, it supports varying degrees of play and gaming. These are some uses of term: MMOGs game: large number of players within a game, and RPG game, etc... \\[0.35cm] To create a virtual world, there're some types of engine that allow you to make like Unity, Unreal Engine, blender, etc... \\[0.35cm] From one of those engines, Unity is the most popular choice for many people, from beginner to expert. \\[0.35cm] Unity is a cross-platform game engine developed by Unity Technologies, first announced and released in June 2005 at Apple Inc.'s Worldwide Developers Conference as a Mac OS X-exclusive game engine. As of 2018, the engine had been extended to support more than 25 platforms. The engine can be used to create three-dimensional, two-dimensional, virtual reality, and augmented reality games, as well as simulations and other experiences. \\[0.35cm]
+ In my opinion, first of all, it has tons of online tutorials to be found, the document with clean format, so a good step for beginners. With it's very intuitive design, C\# language make it easier to use or learn, like you can call other script within a script. Also, with great community, when you have a problem, they'll have you everywhere and every time. Unity has clean API, which is easy to use, understand, implement in C\# code, the application works very well on windows, Linux, android or iOS. Lastly, unlike other engines, the unity has asset store with lots of free assets for everyone to use, while the other ones only have few free assets. \\[0.35cm]In this work, i'll focus on creating the virtual world using unity engine, in particular, it's an RPG game, where you as a player can walk around, interact with other NPC model in the world, or attack some enemy that get in your way. 
```

Comment:
C# language makes it easier -> them s vao make
for instance, you can call other scripts within a script. -> them s vao script, thay like bang for instance. Theo a Hoang em nen thay doi VD nay di. Call other scripts within a script ko phai la dac trung cua C# language. Co the thay bang VD nhu la "for instance, it is very easy to pass functions into other functions as delegates or lamda expressions."
Also, with a great community, when you have problems, they will help you everywhere and every time.
Unity has clean APIs
Lastly, unlike other engines, Unity has an asset store